ex2_system/source/logger/logger.c

```diff
@@ -191,7 +191,7 @@ static void do_output(const char *str) {
     }
 
 #ifndef IS_FLATSAT
-    printf(output_string);
+    printf("%s", output_string);

```

Comment:
What are the subtleties I don't know about this change?

Reply:
This was causing a warning in the unit tests, i.e. when compiling for Linux. I actually looked at replacing printf_ with puts_, but it's not provided.